### PR TITLE
Name changes for 'github.com/Giveth' to 'github.com/spacedecentral' and 'planning-app' to 'planning-suite'

### PR DESCRIPTION
**Pull request by capsulecorplab**
Note: 'login' and 'nameWithOwner' still use 'Giveth' in apps/projects/app/utils/mockData.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ $ npm run start:range
 - Start the dao kit to manage smart contracts interactions between all "planning apps" and aragon official apps (token manager and voting right now)
 - The start script spawns a local blockchain, needed to publish the apps before deploying the dao kit template with all them.
 
-**Detailed information in the [DEVELOPMENT_NOTES.md](https://github.com/Giveth/planning-app/blob/dev/docs/DEVELOPMENT_NOTES.md) document.**
+**Detailed information in the [DEVELOPMENT_NOTES.md](https://github.com/spacedecentral/planning-suite/blob/dev/docs/DEVELOPMENT_NOTES.md) document.**
```
Comment (stellarmagnet) [inline README.md]: should not even put github in the URL to prevent this in the future. the url of /docs/DEVELOPMENT_NOTES.md will work

Comment (capsulecorplab) [reply README.md]: good catch

Comment (stellarmagnet) [reply README.md]: https://help.github.com/articles/about-readmes/#relative-links-and-image-paths-in-readme-files

File: apps/projects/app/utils/mockData.js [hunk 2]
```diff
@@ -603,13 +603,13 @@ const savedRepos = {
...
-                    url: 'https://github.com/Giveth/planning-app/issues/1',
+                    url: 'https://github.com/spacedecentral/planning-suite/issues/1',
                     createdAt: '2018-03-26T23:44:36Z',
                     number: 1,
                     repository: {
                       id: 'MDEwOlJlcG9zaXRvcnkxMjY4OTkxNDM=',
-                      name: 'planning-app',
-                      nameWithOwner: 'Giveth/planning-app',
+                      name: 'planning-suite',
+                      nameWithOwner: 'Giveth/planning-suite',
```
Comment (stellarmagnet) [inline apps/projects/app/utils/mockData.js]: shouldn't this be renamed to spacedecentral/planning-suite

File: apps/projects/app/utils/mockData.js [hunk 3]
```diff
@@ -623,13 +623,13 @@ const savedRepos = {
...
-                    url: 'https://github.com/Giveth/planning-app/issues/2',
+                    url: 'https://github.com/spacedecentral/planning-suite/issues/2',
                     createdAt: '2018-03-26T23:55:20Z',
                     number: 2,
                     repository: {
                       id: 'MDEwOlJlcG9zaXRvcnkxMjY4OTkxNDM=',
-                      name: 'planning-app',
-                      nameWithOwner: 'Giveth/planning-app',
+                      name: 'planning-suite',
+                      nameWithOwner: 'Giveth/planning-suite',
```
Comment (stellarmagnet) [inline apps/projects/app/utils/mockData.js]: spacedecentral

File: apps/projects/app/utils/mockData.js [hunk 4]
```diff
@@ -654,18 +654,18 @@ const savedRepos = {
...
-                    url: 'https://github.com/Giveth/planning-app/issues/3',
+                    url: 'https://github.com/spacedecentral/planning-suite/issues/3',
                     createdAt: '2018-03-26T23:55:40Z',
                     number: 3,
                     repository: {
                       id: 'MDEwOlJlcG9zaXRvcnkxMjY4OTkxNDM=',
-                      name: 'planning-app',
-                      nameWithOwner: 'Giveth/planning-app',
+                      name: 'planning-suite',
+                      nameWithOwner: 'Giveth/planning-suite',
```
Comment (stellarmagnet) [inline apps/projects/app/utils/mockData.js]: spacedecentral

File: apps/projects/app/utils/mockData.js [hunk 5]
```diff
@@ -679,18 +679,18 @@ const savedRepos = {
...
-                    url: 'https://github.com/Giveth/planning-app/issues/4',
+                    url: 'https://github.com/spacedecentral/planning-suite/issues/4',
                     createdAt: '2018-03-26T23:56:10Z',
                     number: 4,
                     repository: {
                       id: 'MDEwOlJlcG9zaXRvcnkxMjY4OTkxNDM=',
-                      name: 'planning-app',
-                      nameWithOwner: 'Giveth/planning-app',
+                      name: 'planning-suite',
+                      nameWithOwner: 'Giveth/planning-suite',
```
Comment (stellarmagnet) [inline apps/projects/app/utils/mockData.js]: spacedecentral

File: apps/projects/app/utils/mockData.js [hunk 6]
```diff
@@ -715,13 +715,13 @@ const savedRepos = {
...
-                    url: 'https://github.com/Giveth/planning-app/issues/5',
+                    url: 'https://github.com/spacedecentral/planning-suite/issues/5',
                     createdAt: '2018-03-26T23:56:20Z',
                     number: 5,
                     repository: {
                       id: 'MDEwOlJlcG9zaXRvcnkxMjY4OTkxNDM=',
-                      name: 'planning-app',
-                      nameWithOwner: 'Giveth/planning-app',
+                      name: 'planning-suite',
+                      nameWithOwner: 'Giveth/planning-suite',
```
Comment (stellarmagnet) [inline apps/projects/app/utils/mockData.js]: spacedecentral

**Comment by stellarmagnet** [comment]
sorry, i didnt say your note - why did namewithowner not change?

**Comment by stellarmagnet** [comment]
also before this is merged, can we double check that everything still works?

**Comment by capsulecorplab** [comment]
> sorry, i didnt say your note - why did namewithowner not change?

I wasn't sure if it'd break things.

**Comment by capsulecorplab** [comment]
okay, I went ahead and changed it anyway. What tests should I run to make sure I didn't break anything?

**Comment by stellarmagnet** [comment]
> > sorry, i didnt say your note - why did namewithowner not change?
> 
> I wasn't sure if it'd break things.

I feel like it'd be good if @ottodevs reviewed the changes to the mockdata file, bc i'm not sure how it currently works with the 'id'. I assume this is some github token method of pulling in mock data?

**Comment by stellarmagnet** [comment]
> okay, I went ahead and changed it anyway. What tests should I run to make sure I didn't break anything?

that the application works? Can we have a TEST_CASE document and include the primary flow in there?

This is the primary flow:
* Open the Allocations App
  * Click the “New Account” button
  * Enter a name and limit, then create the Account
  * The Allocations app should now display a card for the newly created account
  * Use Metamask to transfer ETH into the address of this newly created account (as the Allocation will draw from this balance). Make sure to transfer enough eth to cover the upcoming allocation. (this step can be done after voting, but must be done before executing the vote)
  * Click the context menu for the Account, then select “New Allocation”
  * Enter the Allocation description, then select “Token Transfer”.
  * Input an Amount for the allocation.
  * Enter the options.
    * The options should be valid Ethereum addresses. Click the “+” to include the option.
  * After submitting the Allocation it will forward it to the Range Voting app
* Open the Range Voting App
  * Click on the new range vote
  * Cast a vote
  * After the range vote ends (the time is up), anyone in the DAO can click the “Execute Vote” action which tallies the votes.
* Open the Allocations App
  * Click the context menu for the Account, then select “Execute Payout”
  * This will distribute the tokens/ETH based on the results of the range vote.

**Comment by capsulecorplab** [comment]
> > okay, I went ahead and changed it anyway. What tests should I run to make sure I didn't break anything?
> 
> that the application works? Can we have a TEST_CASE document and include the primary flow for the "Performing an Allocation Range Vote" in there? Please note the current bug though (#236)
> 
> This is the primary flow:
> 
> * Open the Allocations App
>   
>   * Click the “New Account” button
>   * Enter a name and limit, then create the Account
>   * The Allocations app should now display a card for the newly created account
>   * Use Metamask to transfer ETH into the address of this newly created account (as the Allocation will draw from this balance). Make sure to transfer enough eth to cover the upcoming allocation. (this step can be done after voting, but must be done before executing the vote)
>   * Click the context menu for the Account, then select “New Allocation”
>   * Enter the Allocation description, then select “Token Transfer”.
>   * Input an Amount for the allocation.
>   * Enter the options.
>     
>     * The options should be valid Ethereum addresses. Click the “+” to include the option.
>   * After submitting the Allocation it will forward it to the Range Voting app
> * Open the Range Voting App
>   
>   * Click on the new range vote
>   * Cast a vote
>   * After the range vote ends (the time is up), anyone in the DAO can click the “Execute Vote” action which tallies the votes.
> * Open the Allocations App
>   
>   * Click the context menu for the Account, then select “Execute Payout”
>   * This will distribute the tokens/ETH based on the results of the range vote.

Copied the above workflow to #108

**Comment by capsulecorplab** [comment]
Rebased off dev at HEAD 986d318fb433a86cc0793f007ab13e92d918391e to resolve linting and merge conflicts. @ottodevs would you care to pull the merge trigger? :slightly_smiling_face:

File: package.json [hunk 7]
```diff
@@ -37,8 +37,8 @@
     "test:allocations": "cd apps/allocations && npm test",
     "test:projects": "cd apps/projects && npm test",
     "test:range": "cd apps/range-voting && npm test",
-    "lint": "lerna run lint --no-bail && lerna exec --scope=@tpt/apps-* --stream -- npx eslint app",
-    "lint:fix": "lerna exec --scope=@tpt/apps-* --stream -- npx eslint app --fix",
+    "lint": "lerna run lint && lerna exec --scope=@tps/apps-* --stream -- npx eslint app",
```
Comment (capsulecorplab) [inline package.json]: Should we add back `--no-bail`?

**Comment by capsulecorplab** [comment]
Rebased off dev at HEAD 986d318fb433a86cc0793f007ab13e92d918391e to resolve linting and merge conflicts. @ottodevs care to pull the merge trigger? :slightly_smiling_face:

**Pull request merged**